scripts/commands/updateApiDocs.ts

```diff
@@ -245,8 +245,11 @@ async function convertHtmlToMarkdown(
     JSON.stringify(pkg_json, null, 2) + "\n",
   );
 
-  console.log("Saving images");
-  await saveImages(allImages, `${htmlPath}/_images`, pkg);
+  if (await pathExists(`${htmlPath}/_images`)){
```

Comment:
We should only allow this for historical:

```suggestion
  if (!pkg.historical || await pathExists(`${htmlPath}/_images`)){
```